src/js/bookmarkHandler.js

```diff
@@ -90,8 +93,8 @@ const removeChildrens = bookmarkId =>
   });
 
 const getRoot = () =>
-  new Promise(resolve => {
-    // config ?? @dalexanco
+  new Promise(async resolve => {
+    config = await configUtils.load();
```

Comment:
we should provide it using fn params, we probably always already have config when we call this function (background job).